style.css

```diff
@@ -5,21 +5,21 @@
 
```

Comment:
Don´t forget to set the universal styles
```suggestion
*{
    margin: 0;
    padding: 0;
    box-sizing: border-box;
}

```

Reply:
Done